config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :amusement_parks, only: [:show]
+  
+  resources :mechanics, only: [:show] do
+    resources :rides,  only: [:create], controller: "ride_mechanics"
+  end
+  
```

Comment:
I was super glad to see that you created the ride_mechanics controller! Nice thinking on that one!